main.go

```diff
@@ -480,12 +484,32 @@ func CreateTmpResimDir() error {
 	return nil
 }
 
+func DeleteWorkerDir(dir string) error {
```

Comment:
suggestion: Exclude `/tmp/resim/cache` from the deletion so the experience cache can be preserved